gui/src/components/SettingsButton.tsx

```diff
@@ -33,7 +33,7 @@ export function SettingsButton() {
         onClick={() => setShowSettings(true)}
         sx={{
           justifyContent: "flex-start",
-          [theme.breakpoints.down("md")]: {
+          [theme.breakpoints.down("sm")]: {
```

Comment:
I'm assuming this is due to the change I made to the sidebar breakpoints.
would it make sense to add a custom `sidebar` breakpoint to make the dependency more clear?

Reply:
I thought about that, but I'm also not sure. We only have a few breakpoints so I don't think it's a big issue